<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartConfigDialog</class>
 <widget class="QDialog" name="ChartConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Title</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>X title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditXtitle"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Y title:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditYtitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Font</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonChooseAxisTitleFont">
          <property name="text">
           <string>Choose font</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>401</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Label:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonChooseAxisLabelFont">
          <property name="text">
           <string>Choose font</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>401</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Legend:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonChooseLegendFont">
          <property name="text">
           <string>Choose font</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>401</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Color</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Font:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonFontColor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>501</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Axis:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonAxisColor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>501</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>BG:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonBGColor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>501</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Line style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Plot </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxLineStyleNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonColorPlot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBoxLineTypePlot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="lineEditWidthPlot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Xgrid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacerXgrid">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonColorXgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBoxLineTypeXgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEditWidthXgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ygrid</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacerYgrid">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonColorYgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBoxLineTypeYgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEditWidthYgrid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLegend">
     <property name="title">
      <string>Legend position </string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxLegendDraggable">
        <property name="text">
         <string>Draggable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxLegendXpos"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxLegendYpos"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>164</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>77</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Restore default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonChooseAxisTitleFont</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseAxisTitleFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonChooseLegendFont</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseLegendFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonChooseAxisLabelFont</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseAxisLabelFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFontColor</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseFontColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAxisColor</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseAxisColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBGColor</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseBGColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColorPlot</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>choosePlotColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColorXgrid</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseXgridColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColorYgrid</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>chooseYgridColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditWidthPlot</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>changePlotLineWidth(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditWidthXgrid</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>changeXgridWidth(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditWidthYgrid</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>changeYgridWidth(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>setDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditXtitle</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>setXtitle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditYtitle</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>setYtitle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLegendDraggable</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxLegendXpos</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLegendDraggable</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxLegendYpos</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLegendDraggable</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChartConfigDialog</receiver>
   <slot>handleDraggableChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseAxisTitleFont()</slot>
  <slot>chooseAxisLabelFont()</slot>
  <slot>chooseLegendFont()</slot>
  <slot>chooseFontColor()</slot>
  <slot>chooseAxisColor()</slot>
  <slot>chooseBGColor()</slot>
  <slot>choosePlotColor()</slot>
  <slot>chooseXgridColor()</slot>
  <slot>chooseYgridColor()</slot>
  <slot>changePlotLineType(QString)</slot>
  <slot>changeXgridLineType(QString)</slot>
  <slot>changeYgridLineType(QString)</slot>
  <slot>changePlotLineWidth(QString)</slot>
  <slot>changeXgridWidth(QString)</slot>
  <slot>changeYgridWidth(QString)</slot>
  <slot>setDefault()</slot>
  <slot>setXtitle()</slot>
  <slot>setYtitle()</slot>
  <slot>handleDraggableChecked(bool)</slot>
  <slot>changeLegendXpos(QString)</slot>
  <slot>changeLegendYpos(QString)</slot>
  <slot>setShowLegend(bool)</slot>
 </slots>
</ui>
